### ISSUE TITLE
LAist Donation Form: Update the email confirmation

### ISSUE THREAD

**Issue by arellaTV** [SCPR/SCPRv4#1262]
Tracey reached out asking if it was possible to tailor the message coming just from that one Mango page (refId=la2018)

AC:
- [x] Find out if it's possible to tailor message from one Mango page
- [x] If it's easy, implement
~~- [ ] If not, create a new ticket with more details~~
------
~~- [ ] If more than 1 point, implement for all confirmation pages (which is 0.5 points)~~

**Comment by arellaTV** [comment]
Was able to get a custom confirmation email for contribution forms with refId's `la2018`, `la2018culture`, and `la2018news`. Working with membership to get the wording right, so I'll keep this in review until its shipped. Otherwise, the technical portion of this ticket is mostly complete 👍

**Comment by arellaTV** [comment]
Updated the sender and wording a bit by request. Waiting on the creation of the email account before pushing to production

**Comment by arellaTV** [comment]
Still waiting on the creation of membership@laist.com. Currently finding out the progress of that and who's implementing.

**Comment by arellaTV** [comment]
membership@laist.com has been created and the new LAist confirmation email has been pushed to production.
